MultiThreading.Task3.MatrixMultiplier.Tests/MultiplierTest.cs

```diff
@@ -20,6 +25,19 @@ public void ParallelEfficiencyTest()
         {
             // todo: implement a test method to check the size of the matrix which makes parallel multiplication more effective than
             // todo: the regular one
+
+            var timer1 = System.Diagnostics.Stopwatch.StartNew();
```

Comment:
"Find out the size that makes parallel multiplication more effective than the regular one."

Reply:
I have no idea how to do it by writing a test?

Reply:
Write some code in a test that will increase the matrices count and compare the time untill the parralel version will run faster. Then you can simply assert on that it is faster. This is not a classic test by itself but rather a calculation